PARAMETERS_MAPPING/parameters.csv

```diff
@@ -196,4 +196,17 @@ unique_id;cf_standard_name;cf_alias_name;long_name;imos_vocabulary_name;imos_voc
 201;;;hail_intensity_10min_sample;;;
 202;thickness_of_rainfall_amount;;rainfall_amount_since_9am;;;
 203;;;rainfall duration;;;
-204;rainfall_rate;;rainfall intensity_10min_sample;;;
\ No newline at end of file
+204;rainfall_rate;;rainfall intensity_10min_sample;;;
+850;number_concentration_of_biological_taxon_in_sea_water;;total abundance;Abundance of biota;;
+851;;;cell volume;Biovolume;;Mean size of cell
+852;;;length;;;Average total length
```

Comment:
As above, "Average total length" is in the wrong column.
Also, length of what? That needs to be part of the long name

Reply:
yes